### PR TITLE
Eva Dataset code

### PR DESCRIPTION
**Pull request by rsinema**
Dataset/Datamodule classes for the EVA dataset

### CODE REVIEW

File: configs/train/eva/eva_gaussian_beta_scheduler_cfg.yaml
Comment (spencermyoung513) [inline configs/train/eva/eva_gaussian_beta_scheduler_cfg.yaml:1]: Can we rename this (and all other `beta_scheduler` files) to `seitzer`? So `eva_seitzer_cfg.yaml`. Mostly for appearances when we share this repo with reviewers haha. They won't know what beta scheduler means

File: probcal/enums.py [hunk 1]
```diff
@@ -42,6 +42,8 @@ class ImageDatasetName(Enum):
     MNIST = "mnist"
     COCO_PEOPLE = "coco_people"
     AAF = "aaf"
+    OOD_COCO_PEOPLE = "ood_coco_people"
```
Comment (spencermyoung513) [inline probcal/enums.py]: I think this enum value is old and can be dropped. Probably got in on an earlier merge

File: probcal/experiments/eva_experiment.py
Comment (spencermyoung513) [inline probcal/experiments/eva_experiment.py:1]: Is this script what was used to get the eva results? Or did you end up using `eval_model.py`. If the second, no need to include this file in the PR

Comment (rsinema) [reply probcal/experiments/eva_experiment.py:1]: We used eval_model.py, I forgot this file was in here I think we just used it to look at the results before eval_script.py was in the repo. I'll remove it.

File: configs/train/eva/eva_gaussian_beta_scheduler_cfg.yaml
Comment (rsinema) [reply configs/train/eva/eva_gaussian_beta_scheduler_cfg.yaml:1]: Done! Do you want the faithful and natural gaussian to be Strin and Immer too or should those be left as is?

Comment (spencermyoung513) [reply configs/train/eva/eva_gaussian_beta_scheduler_cfg.yaml:1]: Those are more recognizable names from the literature, so we can leave them

**Review by spencermyoung513 — APPROVED** [review]

**Pull request merged**